### PR TITLE
Support for folder filter and nested image files

### PR DESCRIPTION
**Pull request by uweseimet**
This is the new, consolidated PR.

### CODE REVIEW

File: doc/rasctl_man_page.txt [hunk 1]
```diff
@@ -6,8 +6,18 @@ NAME
...
        rasctl -e | -l | -m | -o | -s | -v | -D | -I | -L | -O | -P | -T | -V |
        -X | [-C FILENAME:FILESIZE] [-E FILENAME] [-F  IMAGE_FOLDER]  [-R  CUR‐
+=======
+       rasctl -e | -l | -m | -s | -v | -D | -I | -L | -O | -P | -T | -V | -X |
+       [-C  FILENAME:FILESIZE]  [-E  FILENAME]  [-F  IMAGE_FOLDER]  [-R   CUR‐
+>>>>>>> feature_file_operations_with_folders
+=======
+       rasctl -e | -l | -m | -o | -s | -v | -D | -I | -L | -O | -P | -T | -V |
+       -X | [-C FILENAME:FILESIZE] [-E FILENAME] [-F  IMAGE_FOLDER]  [-R  CUR‐
+>>>>>>> develop
```
Comment (rdmark) [inline doc/rasctl_man_page.txt]: Some unresolved merge conflicts here.
Edit: Fixed it myself.

**Review by rdmark — APPROVED** [review]

**Comment by rdmark** [comment]
@uweseimet After pondering this for a bit, I would propose that we set the default scan level to 1: the root dir and one level deep. For one, I think a new feature like this should be enabled by default, so that it gets noticed, used, and don't rot away. 

I can't imagine very common usecases to have deeper directory nesting than this, however. Plus, there is the risk of overloading clients with huge amounts of data, in particular initially since the Web Interface is very unoptimized and calls SERVER_INFO very frequently (something that I have on my radar to improve in the future.)

Thoughts?

**Comment by bzeiss** [comment]
@rdmark In my use case, I will probably also use one level only, so scan level 1 would be sufficient for most use cases I guess.

**Comment by uweseimet** [comment]
The default is now a depth of 1.

**Pull request merged**